trezorlib/tests/unit_tests/test_tx_api.py

```diff
@@ -42,3 +43,11 @@ def test_tx_api_gettx():
 
     TxApiTestnet.get_tx('6f90f3c7cbec2258b0971056ef3fe34128dbde30daa9c0639a898f9977299d54')
     TxApiTestnet.get_tx('d6da21677d7cca5f42fbc7631d062c9ae918a0254f7c6c22de8e8cb7fd5b8236')
+
+
+def test_tx_api_current_block():
+    TxApiZencash.current_height()
+
+
+def test_tx_api_get_block_hash():
+    TxApiZencash.get_block_hash(110000)  # 110000 - test value
```

Comment:
your tests need some asserts

...i realize the previous test doesn't have any, and it's a bad test for that

you should do something like:
```python
def test_tx_api_current_block():
￼    height = TxApiZencash.current_height()
	assert height > 12345689  # replace this number with today's block height.
￼ 

￼def test_tx_api_get_block_hash():
￼    hash = TxApiZencash.get_block_hash(110000)
	assert hash == binascii.hexlify("klsjadféjěšklasdfasjlkjsa") # replace with block hash 110000
```

you should also add a test for the bip115 fields: make a function `test_tx_api_bip115`, use `tx = TxApiZencash.get_tx("lakdfoiaslwltkj")`, and then `assert` that the fields (`block_height`, `block_hash`) contain correct values.